src/TwilioVideo.ios.js

```diff
@@ -140,12 +140,14 @@ export default class extends Component {
   }
 
   componentWillMount () {
+    TWVideoModule.changeListenerStatus(true)
```

Comment:
Very cool!  What do you think about putting these calls in registerEvents and unregisterEvents directly?

Reply:
Changed.